### PR TITLE
[Documentation] AzureCommunicationUIChat version update

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
## Purpose

* ui chat sample update

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
``

### CODE REVIEW

**Review by palatter — APPROVED** [review]

File: ui-chat/Podfile.lock [hunk 1]
```diff
@@ -1,75 +1,83 @@
...
     - MicrosoftFluentUI/Separator_ios
-  - MicrosoftFluentUI/TouchForwardingView_ios (0.8.9):
+  - MicrosoftFluentUI/TouchForwardingView_ios (0.10.0):
     - MicrosoftFluentUI/Core_ios
-  - Trouter (0.1.0)
+  - Trouter (0.2.0)
 
 DEPENDENCIES:
-  - AzureCommunicationUIChat (= 1.0.0-beta.2)
+  - AzureCommunicationUIChat (from `/Users/viktoriiahuseinova/Documents/projects/communication-ui-library-ios/AzureCommunicationUI/sdk/AzureCommunicationUIChat/AzureCommunicationUIChat.podspec`)
```
Comment (emlynmac) [inline ui-chat/Podfile.lock:54]: Does this want to be updated?

File: ui-chat/Podfile.lock [hunk 2]
```diff
@@ -1,75 +1,83 @@
...
   trunk:
     - AzureCommunicationChat
     - AzureCommunicationCommon
-    - AzureCommunicationUIChat
     - AzureCore
     - MicrosoftFluentUI
     - Trouter
 
+EXTERNAL SOURCES:
+  AzureCommunicationUIChat:
```
Comment (emlynmac) [inline ui-chat/Podfile.lock:65]: Probably don't wan this committed.

File: ui-chat/Podfile.lock [hunk 3]
```diff
@@ -1,75 +1,83 @@
...
   AzureCore: ebbf7cf4dfe72afc7584088c38d1c99f5a35d647
-  MicrosoftFluentUI: a204df7fac4bf62b608ad0f9683a5ca64faec2be
-  Trouter: 62f6db5519829c0897832d404338c3b78ea64fa2
+  MicrosoftFluentUI: f6db695718efb93f4ca9bdc366c00b80a1f91dba
+  Trouter: 7eab36954ad740e32db3ff9f6025b0dff92193d7
 
-PODFILE CHECKSUM: 92ad270da7a7affe0a5161a63c7a401fe3a2ef0f
+PODFILE CHECKSUM: 7ba8ec1f7e06668a261c1c750c4e4daadb5ffa31
 
 COCOAPODS: 1.11.3
```
Comment (emlynmac) [inline ui-chat/Podfile.lock:83]: Cocoapods 1.12 is out.

File: ui-chat/Podfile.lock (hunk 1 above)
Comment (vhuseinova-msft) [reply ui-chat/Podfile.lock:54]: You're right, should be commited but will be updated when pod is installed!

File: ui-chat/Podfile.lock (hunk 2 above)
Comment (vhuseinova-msft) [reply ui-chat/Podfile.lock:65]: will be updated when pod is released!

**Review by emlynmac — APPROVED** [review]

**Pull request merged**